src/utils/text/index.js

```diff
@@ -38,9 +38,18 @@ export const videoTitleText = 'Video'
 
 export const noVideoDescriptionText = 'No video'
 
-export const inProgressTitleText = 'In progress'
+export const inProgressDescriptionText = 'These lessons are being worked on.'
+export const inReviewDescriptionText = 'These lessons are waiting for review to proceed.'
+export const selfReviewDescriptionText = 'Since you have 12+ lessons published, you can review your own lessons.'
+export const inQueueDescriptionText = 'These lessons are in the publishing queue. The queue automatically publishes them from top to bottom - unless they are a part of a course then they wait until the entire course is published.'
+export const finishedDescriptionText = 'These lessons are available publicly for students to view on egghead.io.'
 
+export const availableTitleText = 'Available'
+export const inProgressTitleText = 'In progress'
+export const inReviewTitleText = 'In review'
 export const inQueueTitleText = 'In queue'
+export const finishedTitleText = 'Finished'
```

Comment:
Sounds good. I was trying to find a way to summarize the finished states since the "finished" tab includes `published`, `flagged`, and `revised`. But yeah published sounds better. Will push an update and merge.